### PR TITLE
Lodash types

### PR DESCRIPTION
**Pull request by FifiTheBulldog**
Add types for lodash as `window._`, and add lodash to common modules.

### CODE REVIEW

File: src/globals.d.ts [hunk 1]
```diff
@@ -3,26 +3,31 @@
...
   };
 
   interface Window {
-    React: typeof React;
     RepluggedNative: RepluggedNativeType;
+    DiscordNative: typeof DiscordNative;
     replugged: typeof replugged;
     // eslint-disable-next-line @typescript-eslint/naming-convention
     webpackChunkdiscord_app: WebpackChunkGlobal;
+    _: typeof _;
```
Comment (asportnoy) [inline src/globals.d.ts:19]: Shouldn't this be in common modules

Comment (FifiTheBulldog) [reply src/globals.d.ts:19]: It is, but lodash is available globally and I see no harm in allowing devs to use that instead of an inconvenient import.

Comment (asportnoy) [reply src/globals.d.ts:19]: I think we should do the same with React then. That way ppl can import it directly.

Comment (FifiTheBulldog) [reply src/globals.d.ts:19]: Hmm, maybe, although `window._` is created by Discord while `window.React` would be us polluting the window...out of scope for this though

Comment (asportnoy) [reply src/globals.d.ts:19]: I think it's worth it but we can create a separate PR for that.

**Review by asportnoy — APPROVED** [review]

**Pull request merged**